users/urls.py

```diff
@@ -1,10 +1,32 @@
-from django.conf.urls import url, include
-from rest_framework.routers import DefaultRouter
-from users import views
+from django.conf.urls import url
+from rest_framework import renderers
+from rest_framework.urlpatterns import format_suffix_patterns
+from users.views import UserViewSet
+from users.views import UsersViewSet
+from users.views import api_root
 
-router = DefaultRouter()
-router.register(r'users', views.UserViewSet)
+users_list = UsersViewSet.as_view({
+    'get': 'list',
+    'post': 'create'
+})
 
-urlpatterns = [
-    url(r'^', include(router.urls))
-]
+users_detail = UsersViewSet.as_view({
+    'get': 'retrieve',
+    'put': 'update',
+    'patch': 'partial_update',
+    'delete': 'destroy'
+})
+
+user_detail = UserViewSet.as_view({
+    'get': 'retrieve',
+    'put': 'update',
+    'patch': 'partial_update',
+    'delete': 'destroy'
+})
+
+urlpatterns = format_suffix_patterns([
+    url(r'^$', api_root),
+    url(r'^users/$', users_list, name='users-list'),
+    url(r'^users/(?P<pk>[0-9]+)/$', users_detail, name='users-detail'),
+    url(r'^account/(?P<pk>[0-9]+)/$', user_detail, name='user-detail'),
```

Comment:
`user-detail` and `users-detail` is too close names. Try to use more letters to describe more accurately, what is which view for.